apiary.apib

```diff
@@ -83,24 +83,139 @@ TODO
+
+Outputs a list of peers that were added (that weren't already
+in the bootstrap list).
+
++ Parameters
+    + arg (Multiaddr, required) - A peer to add to the bootstrap list
+    + default (boolean, optional) - add default bootstrap nodes
 
 + Response 200 (text/json)
 
+    + Attributes (object)
+        + Peers (array)
+            - (Multihash)
+
+    + Body
+
+        ```
+        {
+          "Peers": [
+            "/ip4/5.9.33.222/tcp/4001/ipfs/QmNRV7kyUxYaQ4KzxFXPYm8EfuzJbtGn1wSFenjXL6LD8y"
+          ]
+        }
+        ```
+
++ Response 400 (text/json)
+
+    + Attributes (object)
+        + Message: no bootstrap peers to add (string)
+        + Code: 1 (number)
+
+    + Body
+
+        ```
+        {
+          "Message": "no bootstrap peers to add",
+          "Code": 1
+        }
+        ```
+
++ Response 500 (text/json)
+
+    + Attributes (object)
+        + Message: invalid multiaddr, must begin with / (string)
+        + Code: 0 (number)
+
+    + Body
+
+        ```
+        {
+          "Message": "invalid multiaddr, must begin with /",
+          "Code": 0
+        }
+        ```
+
+
 ## list [GET /bootstrap/list]
+Show peers in the bootstrap list
+
+Peers are output in the format '<multiaddr>/<peerID>'.
 
 + Response 200 (text/plain)
 
+    + Attributes (object)
+        + Peers (array)
+            - (Multihash)
+    + Body
+
+        ```
+        {
+        "Peers": [
+            "/ip4/104.131.131.82/tcp/4001/ipfs/QmaCpDMGvV2BGHeYERUEnRQAwe3N8SzbUtfsmvsqQLuvuJ",
+            "/ip4/104.236.176.52/tcp/4001/ipfs/QmSoLnSGccFuZQJzRadHn95W2CrSFmZuTdDWP8HXaHca9z",
+            "/ip4/104.236.179.241/tcp/4001/ipfs/QmSoLpPVmHKQ4XTPdz8tjDFgdeRFkpV8JgYq8JVJ69RrZm",
+            "/ip4/162.243.248.213/tcp/4001/ipfs/QmSoLueR4xBeUbY9WZ9xGUUxunbKWcrNFTDAadQJmocnWm",
+            "/ip4/128.199.219.111/tcp/4001/ipfs/QmSoLSafTMBsPKadTEgaXctDQVcqN88CNLHXMkTNwMKPnu",
+            "/ip4/104.236.76.40/tcp/4001/ipfs/QmSoLV4Bbm51jM9C4gDYZQ9Cy3U6aXMJDAbzgu2fzaDs64",
+            "/ip4/178.62.158.247/tcp/4001/ipfs/QmSoLer265NRgSp2LA3dPaeykiS1J6DifTC88f5uVQKNAd",
+            "/ip4/178.62.61.185/tcp/4001/ipfs/QmSoLMeWqB7YGVLJN3pNLQpmmEk35v6wYtsMGLzSr5QBU3",
+            "/ip4/104.236.151.122/tcp/4001/ipfs/QmSoLju6m7xTh3DuokvT3886QRYqxAzb1kShaanJgW36yx"
+          ]
+        }
+        ```
+
 ## rm [GET /bootstrap/rm]
+Removes peers from the bootstrap list
+
+Outputs the list of peers that were removed.
+
++ Parameters
+    + arg (Multiaddr, optional) - A peer to add to the bootstrap list
+    + all (bool, optional) Remove all bootstrap peers.
 
 + Response 200 (text/plain)
 
-# Group cat
+    + Attributes (object)
+        + Peers (array)
+            - (Multihash)
 
-## cat [GET /cat]
+    + Body
 
-+ Response 200 (text/plain)
+        ```
+        {
+          "Peers": []
+        }
+        ```
+
++ Response 500 (text/json)
+
+    + Attributes (object)
+        + Message: invalid ipfs address (string)
+        + Code: 0 (number)
+
+    + Body
```

Comment:
Don't I specify examples of all attributes for the other methods, too? At what point is the body not redundant?